grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -167,14 +167,25 @@ def get_aoi(vector=None):
     if vector:
         args['input'] = vector
 
+    topology = gs.parse_command('v.info', map=vector, flags='t')
+    if int(topology['areas']) < 1:
+        gs.fatal(_("No areas found in AOI map <{}>...").format(vector))
+    elif  int(topology['areas']) > 1:
+        gs.warning(_("More than one areas found in AOI map <{}>. \
+                      Using only the first area...").format(vector))
+
     # are we in LatLong location?
     s = gs.read_command("g.proj", flags='j')
     kv = gs.parse_key_val(s)
     if '+proj' not in kv:
         gs.fatal('Unable to get AOI: unprojected location not supported')
     geom_dict = gs.parse_command('v.out.ascii', format='wkt', **args)
```

Comment:
In the future I would love to seem pyGRASS to be used more widely. This would a nice showcase. But it's not blocker for this PR.

Reply:
Agreed - the first case I have addressed. More suggestions welcome.